nncf/experimental/onnx/dataloaders/imagenet_dataloader.py

```diff
@@ -35,14 +35,19 @@ def __len__(self):
         return len(self.dataset)
 
 
-def create_dataloader_from_imagenet_torch_dataset(dataset_dir, input_shape: List[int], batch_size: int = 1,
+def create_dataloader_from_imagenet_torch_dataset(dataset_dir: str,
+                                                  input_shape: List[int],
+                                                  mean=(0.485, 0.456, 0.406),
+                                                  std=(0.229, 0.224, 0.225),
+                                                  crop_ratio=0.875,
+                                                  batch_size: int = 1,
                                                   shuffle: bool = True):
```

Comment:
Are we sure that without the `seed` setting, results with the shuffling will be the same from run to run?

Reply:
`seed` is set in creating `Sampler` if we talking about `RandomSampler`.